dlog/commitment/src/commitment.rs

```diff
@@ -364,11 +378,11 @@ impl<G: AffineCurve> SRS<G> {
             Fr<G>,      // scaling factor for polynoms
             Fr<G>,      // scaling factor for evaluation point powers
             Vec<(
-                G,                  // unshifted polycommitment
-                Vec<Fr<G>>,         // vector of evaluations
-                Option<(G, usize)>, // shifted polycommitment and degree bound
+                &PolyComm<G>,       // polycommitment
+                Vec<&Vec<Fr<G>>>,   // vector of evaluations
+                Option<usize>,      // optional degree bound
```

Comment:
Maybe the degree bound should be inside the PolyComm struct along with the optional group element?

Reply:
I struggle to see the benefit.
Upon serialization, PolyComm has what needs to be serialized. Degree bound would be discarded since it is part of the protocol.  Why to carry it around ?